src/OpenTelemetry.Exporter.Instana/Implementation/Transport.cs

```diff
@@ -26,8 +26,9 @@ namespace OpenTelemetry.Exporter.Instana.Implementation;
 
 internal class Transport
 {
-    private const int MultiSpanBufferLimit = 4096000;
-
+    private const int MULTI_SPAN_BUFFER_SIZE = 4096000;
```

Comment:
Could you please explain why this difference between buffer size and used limit?

Reply:
Buffer size is a size of byte array but we should never reach that size, we should stop before, because of that we have limit config.